stores/GlobalPath/GlobalPathSagas.ts

```diff
@@ -0,0 +1,29 @@
+import BaseSaga from "@/utils/BaseSaga";
+import GlobalPathActions from "./GlobalPathActions";
+import { AnyActions } from "redux";
+import { all, call, delay, put } from 'redux-saga/effects'
+import { GlobalPath } from "./GlobalPathTypes";
+import { GlobalPathService } from "./GlobalPathService";
+
+class GlobalPathSagas extends BaseSaga {
+    *[GlobalPathActions.POLL_GLOBALPATH]() {
+        while (true) {
+            try {
+                const globalpath: GlobalPath[] = yield call(GlobalPathService.getGlobalPath);
+                if (globalpath.length > 0) {
+                    yield put({type: GlobalPathActions.REQUEST_GLOBALPATH_SUCCESS, payload: globalpath});
```

Comment:
Sorry could you also change `globalpath` here to camelcase as well? I missed it.  Variables in javascript should be camel case

Also, let's not insert the entire array of GlobalPath documents to `payload`, but just the lastest document. In the case for GlobalPath, only the latest document is relevant because it gives us the most up-to-date path. You can do this by taking the last element from the array, e.g. `globalPath[globalPath.length -1]`